src/openforms/formio/formatters/custom.py

```diff
@@ -26,17 +29,52 @@ def format(self, component: Component, value: list[float]) -> str:
         return ", ".join((str(x) for x in value))
 
 
+class AddressValue(TypedDict):
+    """
+    Redefined, as the BRK structure is converted to snake_case.
+
+    The Formio data is not transformed, and it stays camelCased. So while it's the same
+    data, the shape is different.
+    """
+
+    postcode: str
+    house_number: str
```

Comment:
I am a bit confused here..I thought it would be `houseNumber`.
And are we going to keep `AddressValue` in `contrib/brk/constants` as well?